src/main/java/objects/battle/Skill.java

```diff
@@ -54,4 +59,33 @@ public void setLag(int lag) {
     public void setType(SkillType type) {
         this.type = type;
     }
+
+    /**
+     * Gives a String list representing the corresponding list of type Skills
+     * @param skillList List of type Skill wanted to be turned into a string
+     * @return String of the names of each skill in skillList
+     */
+    public static List<String> toSkillString(List<Skill> skillList) {
```

Comment:
I am not sure about static methods. Maybe confirm with the TA, but I would suggest moving these into another class like SkillHelper
Or just give each skill a toString method which is not static